src/main/features/claim/guards/newFeaturesConsentGuard.ts

```diff
@@ -5,13 +5,18 @@ import { ClaimStoreClient } from 'claims/claimStoreClient'
 import { FeatureToggles } from 'utils/featureToggles'
 import { GuardFactory } from 'response/guards/guardFactory'
 import { Paths as ClaimPaths } from 'claim/paths'
+import { LaunchDarklyClient } from 'shared/clients/launchDarklyClient'
 
 const claimStoreClient = new ClaimStoreClient()
+const featureToggles: FeatureToggles = new FeatureToggles(new LaunchDarklyClient())
 
 export class NewFeaturesConsentGuard {
 
   static requestHandler (): express.RequestHandler {
     return GuardFactory.createAsync(async (req: express.Request, res: express.Response) => {
+      if (featureToggles.isAutoEnrollIntoNewFeatureEnabled()) {
```

Comment:
Should this be an await ? isAutoEnrollIntoNewFeatreEnabled is an async function so.. where ever you call it it should be an async only

Reply:
Acknowledged. Necessary changes are made as we have discussed.